adapters/handlers/rest/configure_api.go

```diff
@@ -462,6 +482,8 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 		// gracefully stop gRPC server
 		grpcServer.GracefulStop()
 
+		sentry.Flush(2 * time.Second)
```

Comment:
shouldn't here be also `if appState.ServerConfig.Config.Sentry.Enabled` check?

Reply:
Yes! Good catch!